docs/docsite/rst/dev_guide/module_lifecycle.rst

```diff
@@ -1,50 +1,118 @@
 .. _module_lifecycle:
 
-**********************************
-The lifecycle of an Ansible module
-**********************************
+********************************************
+The lifecycle of an Ansible module or plugin
+********************************************
 
-Modules in the main Ansible repo have a defined life cycle, from first introduction to final removal. The module life cycle is tied to the `Ansible release cycle <release_cycle>`.
-A module may move through these four states:
+Modules and plugins in the main Ansible repository have a defined life cycle, from first introduction to final removal. The module and plugin life cycle is tied to the `Ansible release cycle <release_cycle>`.
+A module or plugin may move through these four states:
 
-1. When a module is first accepted into Ansible, we consider it in tech preview and will mark it as such in the documentation.
+1. When a module or plugin is first accepted into Ansible, we consider it in tech preview and will mark it as such in the documentation.
```

Comment:
Does this depend on the old ANSIBLE_METADATA to mark it as tech preview? because I don't think that's used anymore (at least not in collections... not sure about ansible/ansible).

Reply:
Yep, that's the part. I think it's also no longer used in ansible/ansible, but as I wrote:

> (It does not remove the part on 'preview'/'stable' in the lifecycle part, which should also be removed/updated/rewritten. Would be glad if someone else can tackle that part :-) )